tests/MongoDB.Driver.Tests/Specifications/crud/tests/write/insertMany.json

```diff
@@ -25,10 +25,10 @@
       },
       "outcome": {
         "result": {
-          "insertedIds": [
-            2,
-            3
-          ]
+          "insertedIds": {
+            "0": 2,
+            "1": 3
+          }
```

Comment:
I gather the runner already supported this format?

Reply:
I believe so. It looks like `InsertMany(...)` is already implemented to support the map format, and when I ran the unit tests after changing the JSON everything passed.